.github/workflows/build.yaml

```
@@ -32,7 +32,7 @@ concurrency:
 jobs:
   verify-code:
     name: Verify code
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Lets make it `ubuntu-latest`
```suggestion
    runs-on: ubuntu-latest
```